src/data/transformPost.ts

```diff
@@ -9,9 +9,14 @@ import {Lang} from '../models/locale';
  * @param postData - post data
  * @param lang - runtime language
  *
+ * @param plugins - YFM plugins list
  * @returns -prepared post
  */
-export const transformPost = (postData: PostData, lang: Lang) => {
+export const transformPost = (
+    postData: PostData,
+    lang: Lang,
+    {plugins}: TransformPostOptions = {},
```

Comment:
I think we need use agrument-object when arguments quantity more than 2, but it is breaking change. I suggest after merge of this PR to make edits in https://github.com/gravity-ui/blog-constructor/pull/137. can you create issue after merge this PR ?

Reply:
Yes,  of course